ludo.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Name=Ludo
+Icon=ludo
+GenericName=A minimalist frontend for emulators
+Exec=ludo
+Terminal=false
+StartupNotify=false
```

Comment:
You may consider to add `StartupWMClass=ludo` ?

[StartupWMClass](https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html)

> If specified, it is known that the application will map at least one window with the given string as its WM class or WM name hint (see the Startup Notification Protocol Specification for more details).

[Source](https://github.com/libretro/RetroArch/pull/12767)

Reply:
Should it be `Ludo` or `ludo`?